apps/aesophia/test/contracts/remote_state.aes

```diff
@@ -0,0 +1,12 @@
+contract RemoteState =
+  record rstate = { i : int, s : string, m : map(int, int) }
```

Comment:
Is it possible to declare the remote state type as a type in the remote contract, i.e. something like:
```
contract StateHandling =
  record state = { i : int, s : string, m : map(int, int) }

contract RemoteState =
  function get(s : StateHandling.state) = s
```

Though totally optional of course.

Reply:
Nope, I can't figure out a way, but that doesn't mean it isn't possible, I didn't write the compiler :-)

Reply:
Tracked as https://www.pivotaltracker.com/story/show/161309782